Failed to build cni package

When selinux is enforced and building cni package through docker run, it is failing with permission issues like below:
```
[root@localhost cni]# make
mkdir -p /root/my_ws/src/k8s.io/kubernetes/build/cni/output
docker run -it -v /root/my_ws/src/k8s.io/kubernetes/build/cni/output:/output golang:1.7.5 /bin/bash -c "\
git clone https://github.com/containernetworking/cni\
&& cd cni \
&& git checkout 07a8a28637e97b22eb8dfe710eeae1344f69d16e \
&& CGO_ENABLED=0 GOOS=linux GOARCH=amd64 ./build \
&& tar -zcvf cni-amd64-07a8a28637e97b22eb8dfe710eeae1344f69d16e.tar.gz bin/ \
&& mv cni-amd64-07a8a28637e97b22eb8dfe710eeae1344f69d16e.tar.gz /output/"
Cloning into 'cni'...
remote: Counting objects: 3464, done.
remote: Compressing objects: 100% (99/99), done.
remote: Total 3464 (delta 24), reused 0 (delta 0), pack-reused 3363
Receiving objects: 100% (3464/3464), 1.61 MiB | 0 bytes/s, done.
Resolving deltas: 100% (1626/1626), done.
Checking connectivity... done.
Note: checking out '07a8a28637e97b22eb8dfe710eeae1344f69d16e'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

HEAD is now at 07a8a28... Merge pull request #294 from dcbw/host-local-ranges
Building API
Building reference CLI
Building plugins
   flannel
   tuning
   bridge
   ipvlan
   loopback
   macvlan
   ptp
   dhcp
   host-local
   noop
bin/
bin/cnitool
bin/flannel
bin/tuning
bin/bridge
bin/ipvlan
bin/loopback
bin/macvlan
bin/ptp
bin/dhcp
bin/host-local
bin/noop
mv: cannot create regular file '/output/cni-amd64-07a8a28637e97b22eb8dfe710eeae1344f69d16e.tar.gz': Permission denied
make: *** [build] Error 1

```
selinux status:

```
[root@localhost cni]# sestatus
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Max kernel policy version:      28
[root@localhost cni]# 
```

docker version:

```
[root@localhost cni]# docker version
Client:
 Version:         1.12.5
 API version:     1.24
 Package version: docker-common-1.12.5-14.el7.x86_64
 Go version:      go1.7.4
 Git commit:      047e51b/1.12.5
 Built:           Wed Jan 11 17:53:20 2017
 OS/Arch:         linux/amd64

Server:
 Version:         1.12.5
 API version:     1.24
 Package version: docker-common-1.12.5-14.el7.x86_64
 Go version:      go1.7.4
 Git commit:      047e51b/1.12.5
 Built:           Wed Jan 11 17:53:20 2017
 OS/Arch:         linux/amd64
[root@localhost cni]# 
```

--- a/build/cni/Makefile
+++ b/build/cni/Makefile
@@ -29,7 +29,7 @@ all: build
 build:
 	mkdir -p $(OUTPUT_DIR)
 
-	docker run -it -v $(OUTPUT_DIR):/output golang:$(GOLANG_VERSION) /bin/bash -c "\
+	docker run -it -v $(OUTPUT_DIR):/output:Z golang:$(GOLANG_VERSION) /bin/bash -c "\
 	git clone https://github.com/containernetworking/cni\
 	&& cd cni \
 	&& git checkout $(CNI_RELEASE) \

